ner_extended/project.yml

```diff
@@ -0,0 +1,251 @@
+title: "Train Danish DaCy NER transformer models on DANSK"
+description: >
+    This project template lets you train a fine-grained Named-Entity Recognition model on the DANSK dataset containing 18 types annotations.
+    It takes care of downloading the corpus as well as training, evaluating, packaging and releasing the model.
+    The template uses one of more of the transformer models which have been downloaded via Huggingface: 
+      - "jonfd/electra-small-nordic" - small,
+      - "NbAiLab/nb-roberta-base-scandi" - medium,
+      - "KennethEnevoldsen/dfm-bert-large-v1-2048bsz-1Msteps" - large
+      
+    You can run from yaml file using
+    spacy project run WORKFLOW/COMMAND
+
+# Variables can be referenced across the project.yml using ${vars.var_name}
+vars:
+  lang: "da"
+  size: "small"
```

Comment:
can I change this when I run the command or do I need to change the file?

Reply:
As it is set up rn, the config needs changing.